packages/fast-components/src/switch/switch.styles.ts

```diff
@@ -0,0 +1,102 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor } from "../styles";
+
+export const SwitchStyles = css`
+    :host([hidden]) {
+        display: none;
+    }
+
+    :host {
+        position: relative;
+        display: inline-flex;
+        align-items: center;
+        outline: none;
+    }
+
+    :host(.disabled) {
+        opacity: var(--disabled-opacity);
+    }
+
+    :host(.disabled) .label,
+    :host(.readonly) .label,
+    :host(.readonly) .switch,
+    :host(.disabled) .switch {
+        cursor: ${disabledCursor};
+    }
+
+    .switch {
+        position: relative;
+        outline: none;
+        width: calc(var(--height-number) * 2px);
+        height: calc(var(--height-number) * 1px);
+        background: var(--neutral-fill-input-rest);
+        border-radius: calc(var(--height-number) * 1px);
+        border: calc(var(--outline-width) * 1px) solid var(--neutral-outline-rest);
+    }
+
+    .switch:hover {
+        background: var(--neutral-fill-input-hover);
+        border-color: var(--neutral-outline-hover);
+    }
+
+    .switch:active {
+        background: var(--neutral-fill-input-active);
+        border-color: var(--neutral-outline-active);
+    }
+
+    :host(:focus) .switch {
+        box-shadow: 0 0 0 1px var(--neutral-focus) inset;
+        border-color: var(--neutral-focus);
+    }
+
+    .status-indicator {
+        position: absolute;
+        height: calc((var(--height-number) - (var(--design-unit) * 2)) * 1px);
+        width: calc((var(--height-number) - (var(--design-unit) * 2)) * 1px);
+        top: calc(var(--design-unit) * 1px);
+        left: calc(var(--design-unit) * 1px);
+        background: var(--neutral-foreground-rest);
+        border-radius: 50%;
+        transition: all 0.2s ease-in-out;
+    }
+
+    .label .status-maessage {
+        font-family: var(--body-font);
+        color: var(--neutral-foreground-rest);
+        cursor: pointer;
+        ${/* Font size is temporary - replace when adaptive typography is figured out */ ""} font-size: calc(1rem + (var(--density) * 2px));
+    }
+
+    .label {
+        ${/* Need to discuss with Brian how HorizontalSpacingNumber can work. https://github.com/microsoft/fast-dna/issues/2766 */ ""} margin-inline-end: calc(var(--design-unit) * 2px + 2px);
+    }
+
+    ::slotted(.content) {
```

Comment:
I think we want to find an approach that doesn't require the author to put a class on slotted content to achieve the correct styling.

Reply:
Would it be fair to say that regardless of what "content" is, you'd want to have the spacing there? If so, you could just use:
```css
::slotted(*) { }
```